src/rest.cpp

```diff
@@ -52,6 +59,15 @@ static RestErr RESTERR(enum HTTPStatusCode status, string message)
     return re;
 }
 
+void CheckRequestString(const string inputString, enum InputType inputType = IT_UNDEF, size_t maxLength = 0)
```

Comment:
Right.
The rest.cpp class was introduced with `using namespace std;` which is somehow inconsistent with other classes/files. This PR just follows the current rest.cpp implementation code style. But i'm pretty sure someone will open a PR to fix this. :)